src/encoded/tests/test_upgrade_pipeline.py

```diff
@@ -70,3 +70,11 @@ def test_pipeline_upgrade_8_9(upgrader, pipeline_8):
     value = upgrader.upgrade('pipeline', pipeline_8, current_version='8', target_version='9')
     assert value['schema_version'] == '9'
     assert value.get('status') == 'released'
+
+def test_pipeline_upgrade_9_10(upgrader, pipeline_8):
+    pipeline_8['schema_version'] = '9'
+    pipeline_8['assay_term_names'] = ['single-nuclei ATAC-seq', 'HiC']
+    value = upgrader.upgrade('pipeline', pipeline_8, target_version='10')
+    assert value['schema_version'] == '10'
+    assert 'single-nucleus ATAC-seq' in value['assay_term_names']
+    assert 'single-nuclei ATAC-seq' not in value['assay_term_names']
```

Comment:
Maybe make sure `HiC` is still there? It should actually `== ['HiC', 'single-nucleus ATAC-seq']`?

Reply:
Yes, that is checked in the most recent code.